torch/testing/_internal/common_methods_invocations.py

```diff
@@ -595,6 +595,17 @@ class OpInfo(object):
     # if None, then the OpInfo specifies no method variant
     inplace_variant: Callable = _NOTHING
 
+    # explicitly specifies the operator(module) variant of the operator
+    # if _NOTHING (default), the operator variant will be autopopulated
+    # if None, then the OpInfo specifies no operator variant
+    operator_variant: Callable = _NOTHING
+
+    # explicitly specifies the inplace operator variant of the
+    # operator if _NOTHING (default), the inplace operator variant
```

Comment:
This is confusing to read to read because it seems like it's saying "the inplace operator variant of the operator if _NOTHING..." but it intends to say:

```
# allows the inplace operator variant of this operation to be specified as follows:
  - if _NOTHING (default), then the OpInfo attempts to discover the variant using its name
  - if None, then the OpInfo explicitly specifies it has no associated inplace operator
  - if a Callable, then that callable should be the inplace operator associated with this operation
```

We could make the same change to `operator_variant`, `inplace_variant`, and `method_variant` above for consistency if you also think that's clearer, @amjames

Reply:
@mruberry I agree that form of the comment is much clearer. I also may add a note here or below where the `inplace_operator_variant` discovery happens indicating why automatic discovery is disabled if the `inplace_variant` is not defined. 

The `operator` module will attempt to help out by implementing the in-place operator using the out-of-place operator and assigning the result to the lhs. This "works" but does not have proper in-place behavior and will fail tests.